dev_sites.txt

```diff
@@ -0,0 +1,22 @@
+dmptest
+dylandev
+nikosdev
+pablodev
+pieterdev
+sagardev
+test-atlas
+test-deimos
+test-iocaste
+test-jupiter
+test-mars
+test-neptune
+test-nix
+test-pandora
+test-phobos
+test-phoebe
+test-pluto
+test-saturn
+test-tavros
+test-titan
+test-uranus
+test-venus
```

Comment:
Keep in mind that all the `test-*` instances have an extra job for syncing data from any other production instance.

https://github.com/greenpeace/planet4-test-venus/blob/develop/.circleci/config.yml#L189

Reply:
Good catch, I added the job and workflow definitions:
https://github.com/greenpeace/planet4-nro-generator/pull/28/files#diff-639a8d421e4f81988cc6186eb6894cfbR377
https://github.com/greenpeace/planet4-nro-generator/pull/28/files#diff-639a8d421e4f81988cc6186eb6894cfbR461

Reply:
As this was the only difference that `test-*` instances had compared to developer instances (`*dev`), I have just included this job for the latter too. This wasn't the case before, but I guess it shouldn't cause issues and could also be useful there.